app/build.gradle.kts

```diff
@@ -114,6 +114,10 @@ ksp {
 }
 
 dependencies {
+    implementation("io.ktor:ktor-client-core:1.6.0")
+    implementation("io.ktor:ktor-client-cio:1.6.0")
```

Comment:
dependencies need to follow the existing format. define dependencies as they are in libs.version.toml, and implementation(...) here needs to reference those


Also, we already have ktor-client-core, why add it agian?